source/api/discovery/0.1/index.md

```diff
@@ -0,0 +1,633 @@
+  ],
+  "id": "https://example.org/activity/page-1",
+  "type": "OrderedCollectionPage",
+  "startIndex": 20,
+  "partOf": {
+  	"id": "https://example.org/activity/top-collection",
+  	"type": "OrderedCollection"
+  },
+  "prev": {
+  	"id": "https://example.org/activity/page-0",
+  	"type": "OrderedCollectionPage"
+  },
+  "next": {
+  	"id": "https://example.org/activity/page-2",
+  	"type": "OrderedCollectionPage"
+  },
+  "orderedItems": [
+     {
+     	"type": "Update",
+     	"object": {
+     		"id": "https://example.org/iiif/1/manifest",
+     		"type": "Manifest"
+     	},
+     	"endTime": "2018-03-10T10:00:00Z"
+     }
+  ]
+}
+```
+
+### 3.3. Activities
+
+The Activities are the means of describing the changes that have occured in the content provider's system.
+
+Content providers _MAY_ publish Activities separately from Ordered Collection Pages, and if so they _MUST_ be at the HTTP(S) URI given in the `id` property of the Activity.
+
+##### id
+
+An identifier for the Activity.
+
+Activities _MAY_ have an `id` property. The value _MUST_ be a string and it _MUST_ be an HTTP(S) URI. The JSON representation of the Activity _MAY_ be available at the URI.
+
+```
+{ "id": "https://example.org/activity/1" }
+```
+
+##### type
+
+The type of Activity. 
+
+This specification uses the types described in the table below.
+
+| Type   | Definition |
+| ------ | ---------- |
+| `Create` | The initial creation of the resource.  Each resource _MUST_ have at most one `Create` Activity in which it is the `object`. |
+| `Update` | Any change to the resource.  In a system that does not distinguish creation from modification, then all changes _MAY_ have the `Update` type. |
+| `Delete` | The deletion of the resource, or its de-publication from the web. Each resource _MUST_ have at most one Activity in which it is the `object`. |
+{: .api-table #table-type-dfn}
+
+Activities _MUST_ have the `type` property. The value _MUST_ be a registered Activity class, and _SHOULD_ be one of `Create`, `Update`, or `Delete`.
+
+```
+{ "type": "Update" }
+```
+
+##### object
+
+The IIIF resource that was affected by the Activity.  It is an implementation decision whether there are separate lists of Activities, one per object type, or a single list with all of the object types combined.
+
+Activities _MUST_ have the `object` property.  The value _MUST_ be a JSON object, with the `id` and `type` properties.  The `id` _MUST_ be an HTTP(S) URI. The `type` _MUST_ be a class defined in the IIIF Presentation API, and _SHOULD_ be one of `Collection`, or `Manifest`.
+
+```
+{
+  "object": {
+  	"id": "http://example.org/iiif/1/manifest",
+  	"type": "Manifest"
+  }
+}
+```
+
+##### endTime
+
+The time at which the Activity was finished. It is up to the implementer to decide whether the Activity includes the publication of the IIIF resource online, or only the internal data modification, but the decision _MUST_ be consistently applied. 
```

Comment:
Should the spec's use for discovery require that the endTime must correspond to a generally available change in the resource published at the object id? How does it otherwise know that a reindex of the published resource right now will pick up a change?

Reply:
I don't think it must be the time at which the publication happened, but it must be published. Does that need to be explicit?